src/pages/OrderPage.tsx

```diff
@@ -104,31 +87,28 @@ const OrderPage = () => {
   };
 
   return (
-    <>
-      <TheHeader />
-      <Main>
-        <FormProvider {...methods}>
-          <Form onSubmit={methods.handleSubmit(onValid)}>
-            <CardSection {...selectedCard} setSelectedCard={setSelectedCard} />
-            <SenderSection />
-            <ReceiverSection />
-            <GiftInformationSection {...gift} />
-            <Button type="submit">
-              {gift.price *
-                watchedReceiver.reduce(
-                  (total, receiver) => total + Number(receiver.quantity || 0),
-                  0,
-                )}
-              원 주문하기
-            </Button>
-          </Form>
-        </FormProvider>
-      </Main>
-    </>
+    <Main>
+      <FormProvider {...methods}>
+        <Form onSubmit={methods.handleSubmit(onValid)}>
+          <CardSection {...selectedCard} setSelectedCard={setSelectedCard} />
+          <SenderSection />
+          <ReceiverSection />
+          <GiftInformationSection {...gift} />
+          <Button type="submit">
+            {gift.price *
+              watchedReceiver.reduce(
+                (total, receiver) => total + Number(receiver.quantity || 0),
+                0,
+              )}
+            원 주문하기
+          </Button>
+        </Form>
+      </FormProvider>
+    </Main>
   );
 };
 
-export default OrderPage;
+export default withUser(withSuspenseBoundary(OrderPage, true));
```

Comment:
중첩이 되어 쓰이는군요 ㅎㅎ
이런 모습도 상상해보세요

```tsx
export default flow(
  OrderPage
  withSuspenseBoundary,
  withUser,
)
```

고차함수를 이용하면 이렇게 구성할 수도 있답니다!

Reply:
HoC로 쓰임을 보여주고 싶다면 이런 모습도 상상할 수 있겠죠?
```suggestion
export default wrappers(
  [withUser, withSuspenseBoundary],
  OrderPage
)
```

Reply:
wrapper 유틸 함수를 받아 여러 hoc를 적용할 수 있도록 한다면 중첩 구조로 작성하는 것보다 가독성이 좋아지겠네요.
반영해보겠습니다!